addons/mass_mailing_crm/views/mailing_mailing_views.xml

```diff
@@ -22,4 +22,15 @@
     		</xpath>
         </field>
     </record>
+
+    <record id="crm_lead_view_search_from_mailing" model="ir.ui.view">
+        <field name="name">crm.lead.view.search.from.mailing</field>
```

Comment:
guidelines : 
```
<record id="crm_lead_view_search" model="ir.ui.view">
    <field name="name">crm.lead.view.search.inherit.mass.mailing</field>
```

Reply:
+1, indeed my previous comment was either a bit Falsy, either the view was primary. If it is not primary, it just improve the existing one, so no need to add a suffix.

Reason:
base view should be called ``crm.crm_lead_view_tree`` (current naming is historic)
inherit view will be called ``mass_mailing_crm.crm_lead_view_tree``
-> we know those are making a single global view to display, automatic prefix in xml ids tells the module in which a specific view is defined

If view is primary, this is a new view. It has to have a name telling its usage. Aka ``crm_lead_view_search_from_mailing`` for example if it is used only when coming from a mailing.